static/js/compose_error.ts

```diff
@@ -9,8 +9,11 @@ export function show(error_html: string, bad_input?: JQuery, alert_class = "aler
         .stop(true)
         .fadeTo(0, 1);
     $("#compose-error-msg").html(error_html);
-    $("#compose-send-button").prop("disabled", false);
-    $("#sending-indicator").hide();
+    // TODO: Replace with compose_ui.hide_compose_spinner() when it is converted to ts.
+    $("#compose-send-button .loader").hide();
+    $("#compose-send-button span").show();
+    $("#compose-send-button").removeClass("disable-btn");
```

Comment:
Why is this a TODO; is it an import cycle sort of thing?  I'm not sure Typescript fixes that, but it's certainly OK to leave behind one of these for follow-up refactoring to resolve the import cycle in a happy way.

Reply:
I cannot import JS module into ts since it is not typed.